data/volunteersMap.json

```diff
@@ -34,4 +34,6 @@
   "0x4ce5F6d60B833bb411Ba6C63b71EeB398F6634AA": "defnlife",
   "0x4be1989E5E5F13bF3D2cE899e7B5551F6E7Bae3c": "nathanmelaku",
   "0x3512F51dD4462cffd2600b44DFf20dA89F74ecD6": "Jamiu"
```

Comment:
A comma is needed at the end of row 36: `"0x3512F51dD4462cffd2600b44DFf20dA89F74ecD6": "Jamiu",`
For reference, this file contains a JSON object. You can test the validity of a JSON object in any browser's console because JavaScript supports them.

Reply:
Added a comma at the end of row 36 and added it to patch-1